js/adapt-contrib-bookmarking.js

```diff
@@ -34,10 +50,36 @@ class Bookmarking extends Backbone.Controller {
       'menuView:ready': this.setupMenu,
       'pageView:preRender': this.setupPage,
       'view:childAdded': this.onChildViewAdded,
-      'view:preRemove': this.onChildViewPreRemove
+      'view:preRemove': this.onChildViewPreRemove,
+      'bookmarking:resume': this.navigateToFurthest
+    });
+    this.listenTo(documentModifications, {
+      'added:bookmarking': this.onAdd,
+      'removed:div.bookmarking': this.onRemove
     });
   }
 
+  onAdd(event) {
+    if (!this.isEnabled) return;
+    const $target = $(event.target);
+    const model = new BookmarkingModel({
+      label: $target.attr('label') || null,
```

Comment:
Will you ever need html in the label? If so:
```html
<bookmarking><span>some html</span></bookmarking>
```
Also needs to be support.

```suggestion
      label: $target.attr('label') || $target.html() || null,
```